.github/workflows/archery.yml

```diff
@@ -30,6 +30,7 @@ on:
 jobs:
 
   archery:
+    if: ${{ !contains(github.event.pull_request.title, '[skip ci]') }}
```

Comment:
I think that  we don't need `${{ }}` in `if`.

Reply:
@kou yeah, according to the GHA documentation we indeed don't need the curly braces, but nothing has been triggered until I added `${{ }}` around the expression...

Reply:
Oh...